src/styles/globals.css

```
@@ -19,110 +19,122 @@ body,
+}
+
+/* Chromium / Safari */
+input[type="number"]::-webkit-inner-spin-button,
+input[type="number"]::-webkit-outer-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
 .poppins-thin {
-  font-family: 'Poppins', sans-serif;
+  font-family: "Poppins", sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)